test/test_transforms_v2_refactored.py

```diff
@@ -228,25 +228,46 @@ def check_functional_kernel_signature_match(functional, *, kernel, input_type):
         assert functional_param == kernel_param
 
 
-def _check_transform_v1_compatibility(transform, input):
+def _check_transform_v1_compatibility(v2_transform_eager, *inputs, rtol, atol):
     """If the transform defines the ``_v1_transform_cls`` attribute, checks if the transform has a public, static
-    ``get_params`` method, is scriptable, and the scripted version can be called without error."""
-    if transform._v1_transform_cls is None:
+    ``get_params`` method that is the v1 equivalent, and the output is close to v1 in eager and scripted mode."""
+    if len(inputs) != 1:
```

Comment:
This looks pretty unexpected. IIUC this should never happen anyway as we raise an error before?